[2024-04-25T18:26:49.618+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-04-25T18:26:49.906+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my_dag.task_a manual__2024-04-25T18:26:46.447385+00:00 [queued]>
[2024-04-25T18:26:49.944+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my_dag.task_a manual__2024-04-25T18:26:46.447385+00:00 [queued]>
[2024-04-25T18:26:49.947+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 4
[2024-04-25T18:26:49.999+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): task_a> on 2024-04-25 18:26:46.447385+00:00
[2024-04-25T18:26:50.018+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'my_dag', 'task_a', 'manual__2024-04-25T18:26:46.447385+00:00', '--job-id', '82', '--raw', '--subdir', 'DAGS_FOLDER/python_op.py', '--cfg-path', '/tmp/tmpxsj0ah5u']
[2024-04-25T18:26:50.023+0000] {standard_task_runner.py:91} INFO - Job 82: Subtask task_a
[2024-04-25T18:26:50.028+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=4600) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-04-25T18:26:50.030+0000] {standard_task_runner.py:63} INFO - Started process 4601 to run task
[2024-04-25T18:26:50.110+0000] {task_command.py:426} INFO - Running <TaskInstance: my_dag.task_a manual__2024-04-25T18:26:46.447385+00:00 [running]> on host f9a12f9b65d4
[2024-04-25T18:26:50.339+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='my_dag' AIRFLOW_CTX_TASK_ID='task_a' AIRFLOW_CTX_EXECUTION_DATE='2024-04-25T18:26:46.447385+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-04-25T18:26:46.447385+00:00'
[2024-04-25T18:26:50.342+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-04-25T18:26:50.390+0000] {logging_mixin.py:188} INFO - Process DONE!
[2024-04-25T18:26:50.390+0000] {logging_mixin.py:188} INFO - Process DONE!
[2024-04-25T18:26:50.391+0000] {logging_mixin.py:188} INFO - Process DONE!
[2024-04-25T18:26:50.392+0000] {logging_mixin.py:188} INFO - Process DONE!
[2024-04-25T18:26:50.393+0000] {logging_mixin.py:188} INFO - Process DONE!
[2024-04-25T18:26:50.393+0000] {logging_mixin.py:188} INFO - Process DONE!
[2024-04-25T18:26:50.394+0000] {logging_mixin.py:188} INFO - Process DONE!
[2024-04-25T18:26:50.394+0000] {logging_mixin.py:188} INFO - Process DONE!
[2024-04-25T18:26:50.395+0000] {logging_mixin.py:188} INFO - Process DONE!
[2024-04-25T18:26:50.395+0000] {logging_mixin.py:188} INFO - Process DONE!
[2024-04-25T18:26:50.396+0000] {logging_mixin.py:188} INFO - Process DONE!
[2024-04-25T18:26:50.397+0000] {logging_mixin.py:188} INFO - Process DONE!
[2024-04-25T18:26:50.397+0000] {logging_mixin.py:188} INFO - Process DONE!
[2024-04-25T18:26:50.398+0000] {logging_mixin.py:188} INFO - Process DONE!
[2024-04-25T18:26:50.399+0000] {logging_mixin.py:188} INFO - Process DONE!
[2024-04-25T18:26:50.402+0000] {logging_mixin.py:188} INFO - Process DONE!
[2024-04-25T18:26:50.404+0000] {logging_mixin.py:188} INFO - Process DONE!
[2024-04-25T18:26:50.405+0000] {logging_mixin.py:188} INFO - Process DONE!
[2024-04-25T18:26:50.406+0000] {logging_mixin.py:188} INFO - Process DONE!
[2024-04-25T18:26:50.407+0000] {logging_mixin.py:188} INFO - Process DONE!
[2024-04-25T18:26:50.409+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-04-25T18:26:50.411+0000] {taskinstance.py:2892} ERROR - Task failed due to SystemExit(1)
[2024-04-25T18:26:50.427+0000] {taskinstance.py:1205} INFO - Marking task as UP_FOR_RETRY. dag_id=my_dag, task_id=task_a, execution_date=20240425T182646, start_date=20240425T182649, end_date=20240425T182650
[2024-04-25T18:26:50.465+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 82 for task task_a (Task failed due to SystemExit(1); 4601)
[2024-04-25T18:26:50.496+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2024-04-25T18:26:50.535+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-04-25T18:26:50.541+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
